concurrent.CompletableFuture.cancel(booleanÂ mayInterruptIfRunning)\nIf not already completed, completes this CompletableFuture with\n a CancellationException.\n